### ISSUE TITLE
Create a new campaign

### ISSUE THREAD

**Issue by Skywalker13** [blupi-games/planetblupi#25]
Create a new original campaign, new scenario... Why not?

**Comment by DavivaD** [comment]
OK...?
But what will happen with original campaign?
It will be removed or what?

**Comment by Skywalker13** [comment]
Yes in 2.0, it's not for "tomorrow"... No the original campaign will stay forever..
